pandas/guide/source/pandas_setup.rst

```diff
@@ -51,6 +51,8 @@ repository:
 
     | ``git remote add upstream https://github.com/pandas-dev/pandas``
 
+(For Window Users: download git for Windows <https://gitforwindows.org/> and run Git Bash in the directory where you want the copy of pandas source code with the same commends as above.)
```

Comment:
I think it'll be easier for user to find it (or to skip it) if instead of the comment in brackets we use in all the comments for Windows the sphinx `.. note::` directive. This will create a (yellow I think) box. If at the beginning of it we have "Windows users" in bold, Linux and Mac users will skip it easily.

Then, I think in Sphins links usually are between back quotes ended with an underscore. Ignore this comment if it renders the way you want.